### ISSUE TITLE
NTR: Flow cytometry: WASP decreased in hematopoietic cells (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8718]
New term request Flow cytometry: WASP decreased in hematopoietic cells (elthat_100429005527): Wiskott-Aldrich syndrome protein  UMLS: 5237023

**Comment by pnrobinson** [comment]
@MickeySegal
is this WAS protein family=WASP?
Is this a clinical assay or a research-only finding? 
Could you provide more context?

**Comment by MickeySegal** [comment]
Seems clinical from https://www.ncbi.nlm.nih.gov/books/NBK1178/ "Absent or decreased intracellular Wiskott-Aldrich syndrome protein (WASP) detection in hematopoietic cells as determined by flow cytometry or western blotting"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000472
- label: Decreased Wiskott-Aldrich syndrome protein concentration in hematopoietic cells
- parents: [Abnormal tissue metabolite concentration (HP:0032243)](https://hpo.jax.org/app/browse/term/HP:0032243)
- pmid: 
- def: The concentration of Wiskott-Aldrich syndrome protein in hematopoietic cells is below the lower limit of normal. This feature is measured by flow cytometry or Western blotting.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8718
- synonyms:
